data-prepper-plugins/kafka-connect-plugins/README.md

```diff
@@ -184,17 +184,26 @@ Snapshot Mode [ref](https://debezium.io/documentation/reference/stable/connector
-| collections     | YES      |          | List\<Collection\> | The collections to ingest CDC data.                                                                            |
-| force_update    | NO       | FALSE    | Boolean      | When restarting or updating a pipeline, whether to force all connectors to update their config even if the connector name already exists. By default, if the connector name exists, the config will not be updated. The connector name is `<topic_prefix>.<table_name>`. |
+| Option         | Required | Default       | Type               | Description                                                                                                                                                                                                                                                              |
+|----------------|----------|---------------|--------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| hostname       | YES      |               | String             | The hostname of MongoDB server.                                                                                                                                                                                                                                                   |
+| port           | NO       | 27017         | String             | The port of MongoDB server.                                                                                                                                                                                                                                                       |
+| ssl            | NO       | FALSE         | Boolean            | Connector will use SSL to connect to MongoDB instances.                                                                                                                                                                                                                  |
+| ingestion_mode | NO       | export_stream | String             | MongoDB ingestion mode. Available options: export_stream, stream, export                                                                                                                                                                                                 |
+| export_config  | NO       |               | ExportConfig       | The Export Config                                                                                                                                                                                                                                           |
```

Comment:
Should this just be `export` to the user? Would align with `dynamodb`

Reply:
It is not really export. there is no export feature from MongoDB like DynamoDB. You are really connect to DB and read the data from it. Calling ingestion makes more sense.